lib/can/include/CANMsg.h

```diff
@@ -21,12 +22,25 @@ class CANMsg : public CANMessage {
 
   /** Creates CAN message with specific content.
    */
-  CANMsg(int _id, const char *_data, char _len = 8, CANType _type = CANData, CANFormat _format = CANStandard)
-      : CANMessage(_id, _data, _len, _type, _format) {}
+  CANMsg(HWBRIDGE::CANID _id, const char *_data, char _len = 8, CANType _type = CANData,
```

Comment:
Are some of these default variables meant to be ever changed? Can you give me some examples of different parameters you might want to have?

Reply:
um im not actually sure. thats a good question. maybe check wikipedia's can bus page?

Reply:
Wondering if maybe these should just be hardcoded instead. I've been told that default variables are less preferable